ProtoGame2D/Code/Game/Code/Paddle.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Paddle.h"
+
+#include "Engine/Core/CoreIncludes.h"
+#include "Engine/Graphics/Renderer.h"
+#include "Engine/Graphics/Shader.h"
+
+Paddle* g_Paddle = nullptr;
+
+Paddle::Paddle()
+{
+}
+
+Paddle::~Paddle()
+{
+}
+
+void Paddle::Update(float deltaseconds)
+{
+	UNUSED(deltaseconds)
+}
+
+void Paddle::Render(Shader shader)
```

Comment:
consistency of the render function does not change across the app this can be a private method. which is called internally inside render function.

Reply:
He means the signature of the function. Btw shader by value is bad